lib/location/location_core.c

```diff
@@ -383,7 +405,36 @@ void location_core_event_cb_pgps_request(const struct gps_pgps_request *request)
 }
 #endif
 
-void location_core_event_cb(const struct location_data *location)
+#if defined(CONFIG_LOCATION_METHOD_CELLULAR_EXTERNAL)
+void location_core_event_cb_cellular_request(struct lte_lc_cells_info *request)
+{
+	struct location_event_data cell_request_event_data;
+
+	cell_request_event_data.id = LOCATION_EVT_NEIGHBOR_CELL_REQUEST;
+	cell_request_event_data.cellular_request = *request;
+	event_handler(&cell_request_event_data);
+}
+
+void location_core_cellular_ready(struct location_data *location_result)
+{
+	LOG_DBG("Location cellular ready notification");
+	if (location_result == NULL) {
+		current_event_data.id = LOCATION_EVT_ERROR;
+	} else {
+		current_event_data.id = LOCATION_EVT_LOCATION;
+	}
+
+	location_result->method = LOCATION_METHOD_CELLULAR;
+
+	current_event_data.location = *location_result;
+
+	k_work_submit_to_queue(
+		location_core_work_queue_get(),
+		&location_event_cb_work);
+}
+#endif
```

Comment:
What are these in this PR?

Reply:
Removing. Will be in a separate PR for ATv2 support